### PR TITLE
add message signing and verification methods

### PR DESCRIPTION
**Pull request by smk762**
coins ref: https://github.com/KomodoPlatform/coins/pull/463
API ref: https://github.com/KomodoPlatform/atomicDEX-API/issues/1220

### CODE REVIEW

**Review by gcharang — APPROVED** [review]

**Comment by gcharang** [comment]
@smk762 will merge once @caglaryucekaya approves

File: docs/basic-docs/atomicdex-api-20-dev/message_signing.md
Comment (caglaryucekaya) [inline docs/basic-docs/atomicdex-api-20-dev/message_signing.md:37]: Description: The signature generated for the message

File: docs/basic-docs/atomicdex-api-20-dev/message_signing.md
Comment (caglaryucekaya) [inline docs/basic-docs/atomicdex-api-20-dev/message_signing.md:4]: https://bitcoin.stackexchange.com/questions/77324/how-are-bitcoin-signed-messages-generated/77325#77325

This link is more relevant for the message prefix

**Review by caglaryucekaya — CHANGES_REQUESTED** [review]

**Pull request merged**
